crates/ethcore/res/chainspec/test/null_morden.json

```diff
@@ -24,7 +24,7 @@
 		"timestamp": "0x00",
 		"parentHash": "0x0000000000000000000000000000000000000000000000000000000000000000",
 		"extraData": "0x",
-		"gasLimit": "0x2fefd8"
```

Comment:
Is there a reason why you would change this?

Reply:
Reverted with the newest commit.